frontend/src/scenes/settings/project/DataAttributes.tsx

```diff
@@ -34,7 +34,8 @@ export function DataAttributes(): JSX.Element {
             </p>
             <div className="space-y-4 max-w-160">
                 <LemonSelectMultiple
-                    mode="multiple-custom"
+                    mode="multiple"
```

Comment:
This feels like a weird component API to me - I wouldn't necessarily reach for the `SelectMultiple` component assuming it allowed me to specify a `mode="single"`.

Would it be better to choose an altogether different name like `LemonInputSelect`. I've also seen this component referred to as `Autocomplete` or `Comboxbox` in a number of other design systems:
https://mui.com/material-ui/react-autocomplete/#combo-box
https://headlessui.com/react/combobox

Reply:
That makes a lot of sense to rename it. I think the component as a whole makes sense but the name is confusing.

I quite `LemonInputSelect` tbh as thats sort of what it is. 

Autocomplete feels off as that isn't even a core function (options are optional).
Combobox sounds weird to me.